tests/bluetooth/test_gatt_connection.py

```diff
@@ -15,12 +15,33 @@
 
 import pytest
 import pytest_asyncio
-from test_helpers import (execute_command, send_JSON_command, subscribe,
-                          wait_for_event)
+from test_helpers import (AnyExtending, execute_command, send_JSON_command,
+                          subscribe, wait_for_event)
 
 from . import disable_simulation, setup_granted_device
 
 
+async def check_gatt_connected(websocket, context_id: str,
```

Comment:
nit: consider making it "is_gatt_connected" returning bool, and move assertion to tests. It should increase readability and assertion error.

Reply:
Done.